Formula/kromium.rb

```diff
@@ -17,14 +17,16 @@ def install
     system "echo hello > src/file1"
     assert_match "file1", shell_output("ls src")
     assert_match "", shell_output("ls dst")
+    dir = Dir.getwd
     (testpath/"test.cue").write <<~EOS
       {
-      SourceBucket: "file://`pwd`/src"
-      DestinationBucket: "file://`pwd`/dst"
-      StateBucket: "file://`pwd`/state"
+      SourceBucket: "file://#{dir}/src"
+      DestinationBucket: "file://#{dir}/dst"
+      StateBucket: "file://#{dir}/state"
       Transforms: [{ Type: "Identity" }]
       }
     EOS
+    system "cp", "test.cue", "/tmp/"
```

Comment:
Why is this needed?

Reply:
removed was testing with this. I really appreciate your adept and quick review on this 🙏